api/v1alpha1/smbshare_types.go

```diff
@@ -117,6 +117,10 @@ type SmbShareStatus struct {
 // +kubebuilder:object:root=true
 // +kubebuilder:subresource:status
 
+// +kubebuilder:printcolumn:JSONPath=`.spec.shareName`,description="Name of the Samba share",name="Share-name",type=string
+// +kubebuilder:printcolumn:JSONPath=`.spec.storage.pvc.path`,description="Connect path for the share within PVC",name="Share-path",type=string
+// +kubebuilder:printcolumn:JSONPath=`.spec.scaling.availabilityMode`,description="Samba installation mode",name="Setup-Mode",type=string
```

Comment:
I think "Availability" or "Availability Mode" would match. For me, "Setup-Mode" doesn't match up with anything else in the config. Ditto for the description.   I think consistency is very important for user-facing strings.

Reply:
I completely agree here.